sdk/communication/communication-job-router/package.json

```diff
@@ -0,0 +1,130 @@
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha --reporter ../../../common/tools/mocha-multi-reporter.js \"dist-test/index.node.js\"",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": "github:Azure/azure-sdk-for-js",
+  "keywords": [
+    "azure",
+    "cloud",
+    "Azure",
+    "communication",
+    "jobrouter"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication/communication-job-router/",
+  "sideEffects": false,
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "dependencies": {
+    "@azure/abort-controller": "^1.0.0",
+    "@azure/communication-common": "^1.1.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-http": "^1.2.0",
+    "@azure/core-client": "^1.0.0",
+    "@azure/core-rest-pipeline": "^1.1.0",
+    "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/logger": "^1.0.0",
+    "events": "^3.0.0",
+    "tslib": "^2.2.0",
+    "@azure/core-paging": "^1.1.1",
+    "uuid": "^8.3.0"
+  },
+  "devDependencies": {
+    "@azure/core-util": "^1.0.0-beta.1",
+    "@azure/communication-identity": "^1.0.0",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure-tools/test-recorder": "^1.0.0",
+    "@microsoft/api-extractor": "^7.18.11",
+    "@rollup/plugin-commonjs": "11.0.2",
+    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-multi-entry": "^3.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "@rollup/plugin-replace": "^2.2.0",
+    "@types/chai": "^4.1.6",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^12.0.0",
+    "@types/sinon": "^9.0.4",
+    "@types/uuid": "^8.0.0",
+    "assert": "^1.4.1",
```

Comment:
Is assert needed? `chai` should be enough. We are trying to remove `assert` from our packages.